js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -1,29 +1,51 @@
-/* eslint-disable max-classes-per-file */
-
 import {
-  Component, JSXComponent,
+  Component, Consumer, JSXComponent,
 } from '@devextreme-generator/declarations';
 /* eslint-disable import/named */
 import LegacyToolbar from '../../../ui/toolbar';
 
 import { DomComponentWrapper } from '../common/dom_component_wrapper';
 import { ToolbarProps } from './toolbar_props';
+import { isDefined, isString } from '../../../core/utils/type';
+import { ConfigContext, ConfigContextValue } from '../../common/config_context';
+import { resolveRtlEnabled } from '../../utils/resolve_rtl';
+
+export const viewFunction = ({ props, rtl, restAttributes }: Toolbar): JSX.Element => {
+  const { items } = props;
+  if (isDefined(items)) {
+    // eslint-disable-next-line @typescript-eslint/prefer-for-of
+    for (let index = 0; index < items.length; index += 1) {
+      const item = items[index];
+      if (!isString(item)) {
```

Comment:
May be we need to expect isPlainObject(item) instead of !isString?

Reply:
Yep, IsObject (not isPlainObject) successfully meets my requirements. Thank you